html-block-and-inline/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Block and Inline</title>
+  </head>
+  <body>
+    <h1><strong>Blue Americana Snuggie&reg; Blanket</strong></h1>
```

Comment:
you can remove the `strong` tags here as the h1 and h2 tags are bold by default.